orc8r/cloud/go/services/orchestrator/servicers/provider_servicer.go

```diff
@@ -40,7 +40,7 @@ func (s *orchestratorStreamProviderServicer) GetUpdates(ctx context.Context, req
 
 	update, err := streamer.GetUpdates(req.GetGatewayId(), req.GetExtraArgs())
 	if err != nil {
-		// Note: return blank err to properly receive EAGAINs from mconfig provider
+		// Note: return blank err to properly receive EAGAIN from mconfig provider
```

Comment:
Yeah this is tracked as T71188848

The eagain pattern was originally added by @emakeev I think, I'm gonna sync with him to understand the context. But basically it's intended to keep the stream open if there are no updates. If the agw doesn't handle this gracefully, I could see where it would end up with lots of open streams

 But yeah basically (a) we need to be communicating "no updates" via the return proto rather than the error, and (b) need to either (always close the stream after sending updates) or (ensure agw handles this correctly + add a sleep in the loop [0])

[0] orc8r/cloud/go/services/streamer/servicers/servicer.go:71